docs/source/readers.rst

```diff
@@ -299,21 +312,23 @@ Step 2 : Define the reader name
 
 
 Since you aim to design a new reader, you can start by copy-pasting
-`the reader_template.py <https://github.com/ltelab/disdrodb/blob/main/disdrodb/L0/readers/reader_template.py>`_
+`the reader_template.py <https://github.com/ltelab/disdrodb/blob/main/disdrodb/l0/readers/reader_template.py>`_
 python file into the relevant ``disdrodb.l0.reader.<READER_DATA_SOURCE>`` directory.
 If the ``<READER_DATA_SOURCE>`` for your reader does not yet exists, create a new directory.
 Then rename the copied `reader_template.py` file with the name of your reader.
 In order to guarantee consistency between readers, it is very important to follow a specific nomenclature.
 Here in after we will refer to the reader name with ``<READER_NAME>``.
 
 .. note::
-    Guidelines for the definition of **<READER_NAME>**:
-    * The <READER_NAME> should corresponds to the name of the <CAMPAIGN_NAME>.
-    * The <READER_NAME> must be defined UPPER CASE, without spaces.
-    * However, if a campaign requires different readers (because of different file formats or sensors), the <READER_NAME>
-    is defined by adding a suffix preceded by an underscore indicating the stations or the sensor for which has been designed.
-    Example: "RELAMPAGO_OTT" and RELAMPAGO_RD80
-    * Have a look at the `pre-implemented DISDRODB readers <https://github.com/ltelab/disdrodb/tree/main/disdrodb/L0/readers>`_ to grasp the terminology.
+    Guidelines for the definition of ``<READER_NAME>``:
+
+    * The ``<READER_NAME>`` should corresponds to the name of the ``<CAMPAIGN_NAME>``.
+
+    * The ``<READER_NAME>`` must be defined UPPER CASE, without spaces.
+
+    * However, if a campaign requires different readers (because of different file formats or sensors), the ``<READER_NAME>`` is defined by adding a suffix preceded by an underscore indicating the stations or the sensor for which has been designed. Example: "RELAMPAGO_OTT" and "RELAMPAGO_RD80"

```

Comment:
```suggestion
    * However, if a campaign requires different readers (because of different file formats or sensors), the ``<READER_NAME>`` is defined by adding a suffix preceded by an underscore indicating the stations or the sensor for which has been designed. Example: ``"RELAMPAGO_OTT"`` and ``"RELAMPAGO_RD80"``

```